src/events/logs.py

```diff
@@ -161,4 +168,16 @@ def get_regex_list():
     regex.append([events.on_participant_initial_peers,
                   r'DDS_DomainParticipantDiscovery_enableI:value of: ' +
                   r'initial_peers="(.+)"'])
+    regex.append([events.on_no_environment_discovery_peers,
+                  r"RTIOsapi_envVarOrFileGet:environment variable " +
```

Comment:
I recommend you to make these expressions general by using a capture group for the env var or file names. Like  `RTIOsapi_envVarOrFileGet: file (\w+) not found"`